src/ast/passes/probe_expansion.cpp

```diff
@@ -1,34 +1,75 @@
 #include "ast/passes/probe_expansion.h"
 
+#include "ast/ast.h"
 #include "ast/visitor.h"
 #include "bpftrace.h"
+#include "log.h"
 #include "util/wildcard.h"
 
 namespace bpftrace::ast {
 
 using ast::ExpansionType;
 
+class ExpansionResult {
```

Comment:
Super nit: An extra class here seems a bit overkill instead of just using a pair or struct to hold these two maps but YMMV

Reply:
> we can delete the copy constructor to ensure a single instance and can return Expansion::NONE for attach points which don't have an entry in the expansions map.

Yeah true. Comment retracted 😄